src/tests/unit/tests/common/react/named-fc.test.tsx

```diff
@@ -9,20 +9,20 @@ describe('NamedFC', () => {
     const MyFC = NamedFC('MyFC', () => <span>TEXT</span>);
 
     it('applies correct displayName', () => {
-        expect(MyFC.displayName).toEqual('MyFC');
+        expect(MyFC.displayName).toMatchSnapshot('MyFC');
     });
 
     it('renders as expected', () => {
-        const rendered = shallow(<MyFC />);
+        const renderResult = render(<MyFC />);
 
-        expect(rendered.debug()).toEqual('<span>\n  TEXT\n</span>');
+        expect(renderResult.asFragment()).toMatchSnapshot();
     });
 
     it('shallow renders with correct displayName', () => {
```

Comment:
This test isn't really meaningful without the `shallow` functionality. We can exclude it.

Reply:
Excluded it.